app/models/spree/order/checkout.rb

```diff
@@ -78,6 +82,7 @@ def self.define_state_machine!
               after_transition to: :delivery, do: :create_tax_charge!
               after_transition to: :resumed,  do: :after_resume
               after_transition to: :canceled, do: :after_cancel
+              after_transition to: :payment, do: :charge_shipping_and_payment_fees!
```

Comment:
I don't understand how this logic can be affected by the CI environment.

Reply:
no one does :joy: